assets/BeefyLogo.tsx

```diff
@@ -30,53 +35,29 @@ export default function Logo({
+              fill={color}
+            />
+            <Path
+              d="M10.935 12.5927C10.9338 12.945 10.6471 13.2298 10.2949 13.2285C9.94258 13.2272 9.65791 12.9405 9.65918 12.5881C9.66049 12.2358 9.94715 11.9511 10.2994 11.9524H10.3017C10.6525 11.9535 10.9362 12.2388 10.935 12.5899V12.5927Z"
+              fill={color}
+            />
           </G>
         </G>
-        <Defs>
-          <ClipPath id="clip0_2622_51492">
-            <Rect width={24} height={24} fill="white" />
-          </ClipPath>
-        </Defs>
-      </Svg>
-    </View>
+      </G>
+      <Defs>
+        <ClipPath id="clip0_2622_51492">
+          <Rect width={24} height={24} fill="white" />
+        </ClipPath>
+      </Defs>
+    </Svg>
   )
 }
-
-const styles = StyleSheet.create({
-  container: {
-    shadowOffset: { width: 0, height: 0 },
-    shadowRadius: 2,
-    shadowOpacity: 1,
-    shadowColor: 'rgba(46, 51, 56, 0.15)',
-  },
-})
```

Comment:
These are values from the [Valora logo](https://github.com/valora-inc/wallet/blob/d536b3e270bfad58f73cb1abbfa0315729095764/src/images/Logo.tsx#L45-L52) so I thought they were copied by mistake.